Realtime/src/commonMain/kotlin/io/github/jan/supabase/realtime/RealtimeExt.kt

```diff
@@ -157,13 +203,13 @@ suspend inline fun <reified Data : Any> RealtimeChannel.postgresSingleDataFlow(
             filter(filter)
         }
         val data = result.decodeAs<Data>()
-        primaryKey.producer(data) to data
+        primaryKeys.producer(data) to data
     } catch (e: UnknownRestException) {
         error("Data matching filter and table name $table not found")
     }
     val changeFlow = postgresChangeFlow<PostgresAction>(schema) {
         this.table = table
-        filter(primaryKey.columnName, FilterOperator.EQ, key)
+        filter(primaryKeys.columnName, FilterOperator.EQ, key)
```

Comment:
This can't work either

Reply:
You're right, there's some work in progress apparently: https://github.com/supabase/realtime-js/issues/97

Reply:
Yea, I think providing multiple PKs doesn't work for the single value flow for now